libfwupd/fwupd-common.c

```diff
@@ -633,20 +778,17 @@ fwupd_guid_from_data (const guint8 *data, gsize datasz, FwupdGuidFlags flags)
 gboolean
 fwupd_guid_is_valid (const gchar *guid)
 {
-	gint rc;
-	uuid_t uu;
 	if (guid == NULL)
 		return FALSE;
-	if (strlen (guid) != 36)
+	if (!fwupd_guid_from_string (guid, NULL, FWUPD_GUID_FLAG_NONE, NULL))
```

Comment:
Don't you want to check what the error was for not being valid?  Or just don't care?

Reply:
I'd say just don't care. I suppose we could add a GError if you think it's a good idea.

Reply:
Agree Since you have a deliberate NULL case just for checking format, seems like needless work and debugging output.